app/api/email/route.tsx

```diff
@@ -0,0 +1,20 @@
+import loopsClient from "@/lib/loops/client";
+import { NextRequest } from "next/server";
+
+export async function POST(req: NextRequest) {
+  const body = await req.json();
+  const email = body.email;
+
+  try {
+    const resp = await loopsClient.createContact(email);
+    return resp;
+  } catch (error) {
+    console.error(error);
+    const message = error instanceof Error ? error.message : "failed";
+    return Response.json({ message }, { status: 400 });
+  }
+}
+
+export const dynamic = "force-dynamic";
```

Comment:
While you’ve set the dynamic and fetchCache flags, ensure these align with the broader caching policy for application.